wallet/src/ui/app/pages/home/nft-details/index.tsx

```diff
@@ -44,29 +44,51 @@ function NFTdetailsContent({
 
     const NFTDetails = (
         <div className={st.nftDetails}>
-            <div className={st.nftItemDetail}>
-                <div className={st.label}>Object ID</div>
-                <div className={st.value}>
-                    <ExplorerLink
-                        type={ExplorerLinkType.object}
-                        objectID={nftObjectID}
-                        title="View on Sui Explorer"
-                        className={st.explorerLink}
-                        showIcon={false}
-                    >
-                        {shortAddress}
-                    </ExplorerLink>
-                </div>
-            </div>
-
-            {fileExtentionType.name !== '' && (
+            <div className={st.nftDetailsInner}>
                 <div className={st.nftItemDetail}>
-                    <div className={st.label}>Media Type</div>
+                    <div className={st.label}>Object ID</div>
                     <div className={st.value}>
-                        {fileExtentionType?.name} {fileExtentionType.type}
+                        <ExplorerLink
+                            type={ExplorerLinkType.object}
+                            objectID={nftObjectID}
+                            title="View on Sui Explorer"
+                            className={st.explorerLink}
+                            showIcon={false}
+                        >
+                            {shortAddress}
+                        </ExplorerLink>
                     </div>
                 </div>
-            )}
+
+                {fileExtentionType.name !== '' && (
+                    <div className={st.nftItemDetail}>
+                        <div className={st.label}>Media Type</div>
+                        <div className={st.value}>
+                            {fileExtentionType?.name} {fileExtentionType.type}
+                        </div>
+                    </div>
+                )}
+                {!!nftFields?.metadata?.fields?.attributes && (
+                    <>
+                        {nftFields.metadata.fields.attributes.fields.keys.map(
```

Comment:
qq: how big this list pf attributes can be? (or is usually?)
Do you mind adding some screenshots in the PR description?

cc: @mystie711

Reply:
@pchrysochoidis 
Usually not more than 3-4 attributes/tags.
I've attached some video too.

Reply:
I would say the typical collection of NFTs will have around 5-10 attributes